### PR TITLE
Cloudops tweaks

### PR DESCRIPTION
**Pull request by mostlygeek**
Changes: 

1. add extra documentation for deployment
1. wrote script to dump emails from dynamodb table (helpful for testing and ya'know for dumping :)

### CODE REVIEW

**Comment by mostlygeek** [comment]
the deploy docs are really for cloudops references... but now the repo works with the `eb` cli tool

File: README.md [hunk 1]
```diff
@@ -35,9 +35,22 @@ Don't forget to set your AWS creds in `.ebenvironment/environment.config`
...
 elastic beanstalk dashboard.
 
+### Using `eb` CLI
+
+- make sure `eb` is configured correctly for environments
+  - `master` = dev
+  - `stage`  = stage
+  - `prod`   = production
+- change to the appropriate branch (master, stage, production)
+- `./node_modules/.bin/gulp build` - to build assets into `public/` directory
```
Comment (pdehaan) [inline README.md]: **PROTIP:** You can change this to [`npm run build`](https://github.com/mozilla/idea-town/blob/56bd0409a788481eb3e3ed655b31d9a728f0663e/package.json#L8) and not have to worry about having **gulp** installed globally or doing the `./node_modules/.bin/` path trick.

File: README.md [hunk 2]
```diff
@@ -35,9 +35,22 @@ Don't forget to set your AWS creds in `.ebenvironment/environment.config`
 point your browser to localhost:3001
 
 ## Deploy
+
+### Manual deploy
+
 You will need to run `npm run bundle` to create the zip and upload it through the
 elastic beanstalk dashboard.
```
Comment (6a68) [inline README.md]: Would you mind adding `brew install awsebcli` to these manual deploy instructions?

File: README.md [hunk 3]
```diff
@@ -35,9 +35,22 @@ Don't forget to set your AWS creds in `.ebenvironment/environment.config`
...
 
+### Using `eb` CLI
+
+- make sure `eb` is configured correctly for environments
```
Comment (6a68) [inline README.md]: I'm kinda confused by the [eb cli docs](http://docs.aws.amazon.com/elasticbeanstalk/latest/dg/eb-cli3-configuration.html). Could you be a little more specific about how to configure `eb` correctly?

**Comment by 6a68** [comment]
@mostlygeek This is awesome, thanks so much! Happy to merge once the docs are slightly tweaked

File: README.md (hunk 3 above)
Comment (mostlygeek) [inline README.md]: you run `eb init`

File: README.md (hunk 2 above)
Comment (mostlygeek) [inline README.md]: installing `eb` is documented as part of the AWS docs for it. They might changed the preferred way so I don't like to duplicate it here.

Comment (pdehaan) [inline README.md]: What about just adding a link to http://docs.aws.amazon.com/elasticbeanstalk/latest/dg/eb-cli3-install.html ?

File: package.json [hunk 4]
```diff
@@ -5,7 +5,7 @@
   "description": "Ideas come to town",
   "main": "server.js",
   "scripts": {
-    "build": "gulp build",
+    "build": "./node_modules/.bin/gulp build",
```
Comment (pdehaan) [inline package.json]: You shouldn't need this. AFAIK, npm adds `./node_modules/.bin/` to the `$PATH` automatically when doing an `npm run {script}` or `npm test`.

File: README.md (hunk 2 above)
Comment (6a68) [inline README.md]: @mostlygeek I'm satisfied with that answer ^_^

**Pull request merged**